etrago/appl.py

```diff
@@ -320,8 +322,8 @@ def run_etrago(args, json_path):
                  by carrier, set upper/lower limit in p.u.
 
      network_clustering_kmeans : dict
-          {'active': True, 'n_clusters': 30, 'n_clusters_gas': 30,
-           'kmeans_busmap': False, 'line_length_factor': 1.25,
+          {'active': True, 'n_clusters': 30, 'cluster_foreign_AC': False,
+           'n_clusters_gas': 30, 'kmeans_busmap': False, 'line_length_factor': 1.25,
```

Comment:
The new argument "cluster_foreign_AC" is part of the default arguments but not described in the text below. Maybe it could be combined with the gas_clustering documentation

Reply:
Description included in adf6dff52902271495153c290ed2c01ba9ed0ce6